Sources/LogtoClient/LogtoClient/LogtoClient+SignIn.swift

```diff
@@ -32,9 +33,43 @@ public extension LogtoClient {
         )
     }
 
+    // TO-DO: error handling
+    // TO-DO: implement full functions
     func signInWithBrowser() {
-        fetchOidcConfig { _, _ in
-            // To implement
+        fetchOidcConfig { [self] oidcConfig, _ in
+            guard let oidcConfig = oidcConfig else {
+                return
+            }
+
+            let scheme = "io.logto.SwiftUI-Demo"
```

Comment:
Maybe we should provide an approach for users to set up this `scheme`.

Reply:
yes, will do in the next pr